negocio/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Categoria(models.Model):
+    nombre = models.CharField(max_length=200)
+
+    def __str__(self):
+        return self.nombre
+
+
+class Producto(models.Model):
+    id = models.IntegerField(primary_key=True)
+    categoria = models.ForeignKey(Categoria, on_delete=models.CASCADE)
+    nombre = models.CharField(max_length=200)
+    marca = models.CharField(max_length=100)
+    tamano = models.CharField(max_length=4)
+    destacado = models.BooleanField(default=False)
```

Comment:
Si queremos marcar 2 tipos de destacados(semanal, y para la lista de destacados) o bien, este campo no puede ser boolean, o hay que hacer un campo para cada tipo de destacado

Reply:
Entiendo. Entonces sumo un campo más entonces:
- principal
- destacado
```suggestion
    principal = models.BooleanField(default=False)
    destacado = models.BooleanField(default=False)
```